frontend/components/SourceTreeItem.cpp

```diff
@@ -497,6 +497,7 @@ void SourceTreeItem::Update(bool force)
 	} else if (type == Type::Group) {
 		expand = new QCheckBox();
 		expand->setProperty("class", "checkbox-icon indicator-expand");
+		expand->setSizePolicy(QSizePolicy::Maximum, QSizePolicy::Preferred);
```

Comment:
I would use Maximum for both width and height

Reply:
Changed to Maximum for both. Thanks!